### PR TITLE
Introducing Cradle storyline player.

### PR DESCRIPTION
**Pull request by richardlalancette**
- See https://github.com/daterre/Cradle for details.
- Cradle stories are created in Twine online or using the desktop applications.
- Once a story is created, it can be tested using the desktop applications.
- Once tested, the story .html file can be dragged into a story folder in Unity and cradle will generate the story .cs file for us.

@dbarton-aoe Do you want the .src folder committed?
There is a lot of files in this commit due to the source files being present.

Twine Desktop App example
image

One single passage example. This passage is called Landing and is from War of champion PoC.
image

And an example of passage editing:
image

### CODE REVIEW

**Comment by dbarton-aoe** [comment]
Very cool 👍 . Good question, re: src folder. Is this something we can use as a standalone tool, or is it better that we keep it intact with the rest of our source files? If we aren't worried about keeping it up-to-date on a regular basis, I think it makes sense to put any tools on the Google Drive and leave an entry in the read me; otherwise, I think it's fine to merge it into the project if it simplifies use.

**Comment by richardlalancette** [comment]
We are not going to be updating cradle, very unlikely.
So I can remove all of that.

Richard Lalancette

File: UnityProject/Assets/PrologueStoryController.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+﻿using Cradle;
+using UnityEngine;
+using StorylineSystem;
+
+public class PrologueStoryController : MonoBehaviour
```
Comment (richardlalancette) [inline UnityProject/Assets/PrologueStoryController.cs:5]: We can rename this to something more generic later. (StoryController?)

But, this is the gist of it. Add a controller of some sort and connect to the player events. Et voila!

File: UnityProject/Assets/PrologueStoryController.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        _storyPlayer.OnOutput += StoryPlayerOnOnOutput;
+        _storyPlayer.OnOutputRemoved += StoryPlayerOnOnOutputRemoved;
+        
+        if (_storyPlayer.story)
+        {
+            _storyPlayer.story.Begin();
+        }
+    }
+
+    private void StoryPlayerOnOnPassageEnter(StoryPassage storyPassage)
```
Comment (richardlalancette) [inline UnityProject/Assets/PrologueStoryController.cs:27]: Next  part, going to implement these.

**Pull request merged**